[2020-12-09 00:00:13,954] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: Airflow_task.Data_from_table1 2020-12-08T18:25:00+00:00 [queued]>
[2020-12-09 00:00:13,962] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: Airflow_task.Data_from_table1 2020-12-08T18:25:00+00:00 [queued]>
[2020-12-09 00:00:13,962] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2020-12-09 00:00:13,962] {taskinstance.py:881} INFO - Starting attempt 1 of 1
[2020-12-09 00:00:13,962] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2020-12-09 00:00:13,970] {taskinstance.py:901} INFO - Executing <Task(PythonOperator): Data_from_table1> on 2020-12-08T18:25:00+00:00
[2020-12-09 00:00:13,972] {standard_task_runner.py:54} INFO - Started process 23396 to run task
[2020-12-09 00:00:14,017] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', 'Airflow_task', 'Data_from_table1', '2020-12-08T18:25:00+00:00', '--job_id', '320', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/server-elastic.py', '--cfg_path', '/var/folders/x0/npyjyyg15zj3krdkh0pv612w0000gn/T/tmp1us1s1i5']
[2020-12-09 00:00:14,020] {standard_task_runner.py:78} INFO - Job 320: Subtask Data_from_table1
[2020-12-09 00:00:14,055] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: Airflow_task.Data_from_table1 2020-12-08T18:25:00+00:00 [running]> 153.225.168.192.in-addr.arpa
[2020-12-09 00:00:16,550] {conn.py:380} INFO - <BrokerConnection node_id=bootstrap-0 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: connecting to localhost:9092 [('::1', 9092, 0, 0) IPv6]
[2020-12-09 00:00:16,550] {conn.py:1205} INFO - Probing node bootstrap-0 broker version
[2020-12-09 00:00:16,551] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=bootstrap-0 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]> returned error 61. Disconnecting.
[2020-12-09 00:00:16,551] {conn.py:919} INFO - <BrokerConnection node_id=bootstrap-0 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: Closing connection. KafkaConnectionError: 61 ECONNREFUSED
[2020-12-09 00:00:16,599] {conn.py:380} INFO - <BrokerConnection node_id=bootstrap-0 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: connecting to localhost:9092 [('::1', 9092, 0, 0) IPv6]
[2020-12-09 00:00:16,600] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=bootstrap-0 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]> returned error 61. Disconnecting.
[2020-12-09 00:00:16,600] {conn.py:919} INFO - <BrokerConnection node_id=bootstrap-0 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: Closing connection. KafkaConnectionError: 61 ECONNREFUSED
[2020-12-09 00:00:16,649] {conn.py:380} INFO - <BrokerConnection node_id=bootstrap-0 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: connecting to localhost:9092 [('127.0.0.1', 9092) IPv4]
[2020-12-09 00:00:16,650] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=bootstrap-0 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]> returned error 61. Disconnecting.
[2020-12-09 00:00:16,650] {conn.py:919} INFO - <BrokerConnection node_id=bootstrap-0 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: Closing connection. KafkaConnectionError: 61 ECONNREFUSED
[2020-12-09 00:00:16,650] {logging_mixin.py:112} INFO - Error::::: NoBrokersAvailable
[2020-12-09 00:00:16,651] {python_operator.py:114} INFO - Done. Returned value was: None
[2020-12-09 00:00:16,654] {taskinstance.py:1057} INFO - Marking task as SUCCESS.dag_id=Airflow_task, task_id=Data_from_table1, execution_date=20201208T182500, start_date=20201208T183013, end_date=20201208T183016
[2020-12-09 00:00:18,947] {local_task_job.py:102} INFO - Task exited with return code 0
